### PR TITLE
add google search console to the base.html's header tag

### PR DESCRIPTION
**Pull request by sewagodimo**

### CODE REVIEW

File: molo/core/templates/base.html [hunk 1]
```diff
@@ -27,6 +27,9 @@
     <meta property="og:image" content="{% if self.social_media_image %}{{request.site.root_url}}{{ tmp_photo.url }}{% endif %}"/>
     <meta property="og:image:url" content="{% if self.social_media_image %}{{request.site.root_url}}{{ tmp_photo.url }}{% endif %}"/>
     <meta name="viewport" content="width=device-width, initial-scale=1" />
+    {% if settings.core.SiteSettings.google_search_console %}
+    <meta name="google-site-verification" content="{{settings.core.SiteSettings.global_google_search_console}}" />
```
Comment (KaitCrawford) [inline molo/core/templates/base.html]: In the If statement you check for `.google_search_console` but here you're using `.global_google_search_console` what's the difference?

Comment (sewagodimo) [reply molo/core/templates/base.html]: Hey, I seem to have made a mistake in the naming. 
Thank you

File: molo/core/models.py [hunk 2]
```diff
@@ -141,6 +141,15 @@ class SiteSettings(BaseSetting):
...
             " to view analytics on more than one site globally")
     )
+    google_search_console = models.CharField(
+        verbose_name=_('Google Search Console'),
+        max_length=255,
+        null=True,
+        blank=True,
+        help_text=_(
+            "The Google Search Console tracking code (e.g GTM-XXX) to be used"
+            " to view search performance")
```
Comment (KaitCrawford) [inline molo/core/models.py]: Are you sure this Help text is correct? Perhaps I'm misunderstanding the ticket but I thought you needed to add a verification code not a tracking code? The one I added to the corporate site didn't start with `GTM-...`

Comment (sewagodimo) [reply molo/core/models.py]: @KaitCrawford I updated the help text

**Pull request merged**